hack/clean.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2017 Red Hat, Inc.
+#
+
+source hack/common.sh
+
```

Comment:
Need to let the operators cleanup their components before we delete the operators.  I don't think the HCO is capable of cleaning up component operator CRs yet, so we'll have to remove all the CRs individually.
```bash
"${CMD}" delete -f <cdi-cr.yaml>
"${CMD}" delete -f <kubevirt-cr.yaml>
"${CMD}" delete -f <hco-cr.yaml>

# Wait for all crs to be gone
```

Reply:
I tested this yesterday.  The hco will remove the component CRs.  I think you only need to remove the hco-cr and hco-crd

Reply:
Once the hco-crd is removed the hco-cr is removed as well.